django/contrib/admin/sites.py

```diff
@@ -401,6 +413,20 @@ def login(self, request, extra_context=None):
         request.current_app = self.name
         return LoginView.as_view(**defaults)(request)
 
+    @no_append_slash
+    def catch_all_view(self, request):
+        if (settings.APPEND_SLASH and self.append_slash and not request.path_info.endswith('/')):
+            urlconf = getattr(request, 'urlconf', None)
+            path = '%s/' % request.path_info
```

Comment:
```suggestion
            path = '%s/' % request.path
```
:thinking:

Reply:
I'm not sure there's much practical difference here, but `CommonMiddleware` uses `path_info`, so I'd say let's keep that.